src/python/pants/option/parser.py

```diff
@@ -573,7 +573,13 @@ def expand(val_or_str):
                 else:
                     fromfile = val_or_str[1:]
                     try:
-                        with open(fromfile) as fp:
+                        fromfile_path = Path(fromfile)
+                        fromfile_path = (
+                            fromfile_path
+                            if fromfile_path.is_absolute()
+                            else Path(get_buildroot(), fromfile)
+                        )
+                        with fromfile_path.open() as fp:
                             s = fp.read()
```

Comment:
FYI you can use `fromfile_path.read_text()` and get rid of the context manager/nesting :)

Reply:
I thought about that... I don't remember why I didn't...